src/models/modules/bricks.py

```diff
@@ -1 +1,211 @@
 # Low level modules like ConvBnAct
+
+""" PyTorch Conditionally Parameterized Convolution (CondConv)
+
+Paper: CondConv: Conditionally Parameterized Convolutions for Efficient Inference
+(https://arxiv.org/abs/1904.04971)
+
+Hacked together by / Copyright 2020 Ross Wightman
+"""
+
+import math
+from functools import partial
+import numpy as np
+import torch
+from torch import nn as nn
+from torch.nn import functional as F
+
+from .utils.helpers import to_2tuple
+from .conv2d_same import conv2d_same
+from .utils.padding import get_padding_value
+
+
+def get_condconv_initializer(initializer, num_experts, expert_shape):
```

Comment:
Documentation. And check if it possible to use python @. 
https://realpython.com/primer-on-python-decorators/#:~:text=%40my_decorator%0Adef%20say_whee()%3A%0A%20%20%20%20print(%22Whee!%22)